test/stac/stac.ts

```diff
@@ -144,17 +145,22 @@ describe('STAC catalog route', function () {
           expect(catalog.description).to.equal('Harmony output for http://example.com/harmony?job=completedJob');
           expect(catalog.id).to.equal(completedJob.requestId);
           expect(catalog.links).to.eql([
-            { href: '.', rel: 'self', title: 'self' },
             { href: '.', rel: 'root', title: 'root' },
             { href: './0', rel: 'item' },
+            {
+              href: `http://127.0.0.1:4000/stac/${completedJob.requestId}?page=1&limit=${env.defaultResultPageSize}`,
```

Comment:
This is relying on the environment. We should either construct from envs or use a regex when comparing the href. It looks like trying to use a regex might be a pain. I'm not sure if we have a frontendHost env var - can you use `env.backendHost` and env.frontendPort`?

Reply:
Is this better? https://github.com/nasa/harmony/pull/42/commits/e2352b8d2f866b19d58297d1332c65be39582e34